framework/utils/mathUtils.py

```diff
@@ -70,6 +71,10 @@ def createInterp(x, y, lowFill, highFill, kind='linear'):
      @ In, kind, string, optional, interpolation type (default=linear)
      @ Out, interp, function(float) returns float, an interpolation function that takes a single float value and return its interpolated value using lowFill or highFill when the input value is outside of the interpolation range.
   """
+  sv = str(scipy.__version__).split('.')
+  if int(sv[0])> 0 or int(sv[1]) >= 17:
```

Comment:
why is `or` here? should it be `and`?

Reply:
It is ```or``` so that when version 1.0 comes out it will still be true.

Reply:
Got it, thanks.